knowledge_graph/routes.py

```diff
@@ -152,6 +168,29 @@ def receive_user_scores() -> Tuple[Response, int]:
 
     response = Response(dumps(value_scores))
     return response, 200
+    
+
+@app.route('/personal_values', methods=['GET'])
+def get_personal_values():
+    """ Given a session-id, this returns the top three personal values for a user
+    
+    """
+    session_id = int(request.args.get('session-id'))
+    
+    try:
+        scores = db.session.query(Scores).filter_by(session_id=1).first()
+    except:
```

Comment:
Is it worth specifying some exception types here? For instance- we could return invalid session ID for one exception, but internal server error for others? That might make diagnostics a bit easier in the future

Reply:
Sure, I'll add some exceptions

Reply:
I'm not seeing these updated exceptions here- there's just a bare 'except'. Am I just not seeing them for some reason?